src/util/validate-user.js

```diff
@@ -25,3 +28,26 @@ export default function validateUser(req, res) {
 
   return session;
 }
+
+export function validateAuthHeader(req) {
+  if (config.trustedProxies.length == 0) {
+    return true;
+  }
+
+  let sender = proxyaddr(req, 'uniquelocal');
+  let sender_ip = ipaddr.parse(sender);
```

Comment:
```suggestion
  let sender_ip = ipaddr.process(sender);
```

Caught this during my testing. My own setup apparently was using mapped ips, so process was needed to unmap them.